src/containers/LoginForm.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Container,
+  Form,
+  OverlayTrigger,
+  Tooltip,
+  Button,
+} from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+// font-family: TwitterChirp, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+
+const OnHoverClose = styled.div`
+  :hover {
+    background: rgb(15, 20, 25, 0.1);
+    transition-duration: 0.2s;
+  }
+`;
+
+interface MyForm {
+  userName: string;
+  email: string;
+  dateOfBirth: Date;
+  password: string;
+  confirmPassword: string;
+}
+
+export const LoginForm = () => {
+  const { register, handleSubmit } = useForm<MyForm>();
+
+  const submitForm = (data: MyForm) => {
+    return data;
+    // console.log(data);
+  };
+
+  return (
+    <Container
+      style={{ height: 650, width: 600 }}
+      className=" rounded-4  p-2   "
+    >
+      <header className="d-flex justify-content-between align-items-center">
+        <OverlayTrigger
+          key={"bottom"}
+          placement={"bottom"}
+          overlay={<Tooltip id={`tooltip-${"bottom"}`}>Close</Tooltip>}
+        >
+          <OnHoverClose className="p-2 d-flex justify-content-center align-items-center rounded-5">
+            <svg
+              viewBox="0 0 24 24"
+              aria-hidden="true"
+              width="20px"
+              height="20px"
+            >
+              <g>
+                <path d="M13.414 12l5.793-5.793c.39-.39.39-1.023 0-1.414s-1.023-.39-1.414 0L12 10.586 6.207 4.793c-.39-.39-1.023-.39-1.414 0s-.39 1.023 0 1.414L10.586 12l-5.793 5.793c-.39.39-.39 1.023 0 1.414.195.195.45.293.707.293s.512-.098.707-.293L12 13.414l5.793 5.793c.195.195.45.293.707.293s.512-.098.707-.293c.39-.39.39-1.023 0-1.414L13.414 12z" />
+              </g>
+            </svg>
```

Comment:
Instead, you can use an icons from a third-party library

Reply:
This will be realized after someone else adds this library in another task